packages/shared/src/components/plus/PlusList.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactElement } from 'react';
+import { ChecklistBIcon } from '../icons';
+import {
+  Typography,
+  TypographyColor,
+  TypographyTag,
+  TypographyType,
+} from '../typography/Typography';
+import { IconSize } from '../Icon';
+
+const PlusItem = ({ title }: { title: string }) => {
+  return (
+    <li className="flex gap-2">
+      <ChecklistBIcon className="text-text-tertiary" size={IconSize.Small} />{' '}
```

Comment:
```suggestion
      <ChecklistBIcon className="text-text-tertiary" size={IconSize.Small} />
```
This empty space needed?